webapp/tests/mocha/unit/enketo/medic-xpath-extensions.spec.js

```diff
@@ -199,14 +205,18 @@ describe('medic-xpath-extensions', function () {
 
   describe('#to-bikram-sambat()', () => {
     [
-      [{ t: 'str', v: '2015-9-2' }, '१६ भदौ २०७२'],
-      [{ t: 'str', v: '1999-12-12' }, '२६ मंसिर २०५६'],
-      [{ t: 'date', v: new Date('2015-10-01T00:00:00.000') }, '१४ असोज २०७२'],
-      [{ t: 'num', v: 11111 }, '२१ जेठ २०५७'],
-      [{ t: 'arr', v: [{ textContent: '2014-09-22' }] }, '६ असोज २०७१'],
-    ].forEach(([date, expected]) => {
+      [{ t: 'str', v: '2015-9-2' }, '१६ भदौ २०७२', '१५ भदौ २०७२'],
+      [{ t: 'str', v: '1999-12-12' }, '२६ मंसिर २०५६', '२५ मंसिर २०५६'],
+      [{ t: 'date', v: new Date('2015-10-01T00:00:00.000') }, '१४ असोज २०७२', '१३ असोज २०७२'],
+      [{ t: 'num', v: 11111 }, '२१ जेठ २०५७', '२० जेठ २०५७'],
+      [{ t: 'arr', v: [{ textContent: '2014-09-22' }] }, '६ असोज २०७१', '५ असोज २०७१'],
+    ].forEach(([date, expected, offsetExpected]) => {
       it(`should format the ${date.t} [${JSON.stringify(date.v)}] according to the Bikram Sambat calendar`, () => {
-        assert.equal(func['to-bikram-sambat'](date).v, expected);
+        // The resolved Bikram Sambat date for input that does not have a time+TZ value will be different in
+        // environments where the timezone offset is 12 hours or greater (e.g. `Pacific/Auckland`)
```

Comment:
See the [slack thread](https://medic.slack.com/archives/C024KTGRW/p1713461190238749) for more context about these changes....